services/app/src/components/team/Team.jsx

```diff
@@ -7,6 +7,7 @@ import GameInfo from '../gameStatus/GameInfo';
 import playerPropType from '../../prop-types/player';
 import Return from './teamComponents/Return.svg';
 import './team.css';
+import '../../static/style/scrollbar.css'
```

Comment:
These styles are app-wide styles, so they should be imported somewhere near the root of component tree. They are not `Team` component related only, please move this import somewhere near the root of the component tree.